.gitignore

```diff
@@ -10,5 +14,327 @@ gradle-app.setting
 # Cache of project
 .gradletasknamecache
 
-# # Work around https://youtrack.jetbrains.com/issue/IDEA-116898
-# gradle/wrapper/gradle-wrapper.properties
+
```

Comment:
I don't know if it makes sense to include everything out of the box into gitignore. I would approach this as on-request, add entry when it's needed. There's no one gitignore that suits everything and is rather project specific. Current project does not rely on anything else than Dockerfile and Helm chart so I would just list common ignored files according to these two only.

As the code base increases then such ignores (with stuff that's not relevant) makes it more confusing - sometimes you can use gitignore to find out the results of some steps that are recreatable, builds, caches, etc, and gives you understanding that you need to do something to create these ignored files

Reply:
OK. I’ll remove.

Reply:
@thenouan I kept only rules related to OS and IDE. Let me know if it looks OK to you now.